tornado/test/testing_test.py

```diff
@@ -75,6 +76,28 @@ def fail(message):
         self.assertEqual(str(cm.exception), "error one")
 
 
+class AsyncHTTPTestCaseTest(AsyncHTTPTestCase):
+    def get_app(self):
+        return Application()
+
+    def test_fetch_segment(self):
+        path = '/path'
+        response = self.fetch(path)
+        self.assertEqual(response.request.url, self.get_url(path))
+
+    def test_fetch_full_http_url(self):
+        path = 'http://localhost/path'
```

Comment:
This test case should not be using port 80 if it is already in use. `AsyncHTTPTestCase` binds to an unused port by default.
https://github.com/tornadoweb/tornado/blob/master/tornado/testing.py#L376
https://github.com/tornadoweb/tornado/blob/master/tornado/testing.py#L419
https://github.com/tornadoweb/tornado/blob/master/tornado/testing.py#L424-L427